tests/partitions.t

```diff
@@ -0,0 +1,33 @@
+Partitions command success:
+
+  $  ringpop-admin partitions 127.0.0.1:3000
+   Checksum     # Nodes   Sample Host    
+   *   5         127.0.0.1:3000  (glob)
+
+
+With bootstrap file:
+
+  $  ringpop-admin partitions $TESTDIR/hosts.json
+   Checksum     # Nodes   Sample Host    
+   *   5         127.0.0.1:3000  (glob)
+
+
+Unable to connect to host:
+
+  $  ringpop-admin partitions 127.0.0.1:2999
```

Comment:
My tests failed because of this so I think the test is a bit too strict like this